File: src/main/java/eu/mshade/enderframe/item/metadata/ColorItemStackMetadata.java
package eu.mshade.enderframe.item.metadata;

import eu.mshade.enderframe.metadata.MetadataKeyValue;
import eu.mshade.enderframe.metadata.itemstack.ItemStackMetadataKey;
import eu.mshade.enderframe.mojang.Color;

public class ColorItemStackMetadata extends MetadataKeyValue<Color> {

    public ColorItemStackMetadata(Color metadataValue) {
        super(ItemStackMetadataKey.COLOR, metadataValue);
    }



}

File: src/main/java/eu/mshade/enderframe/item/metadata/ColorItemStackMetadata.kt
package eu.mshade.enderframe.item.metadata

import eu.mshade.enderframe.item.ItemStackMetadataKey
import eu.mshade.enderframe.metadata.MetadataKeyValue
import eu.mshade.enderframe.mojang.Color

class ColorItemStackMetadata(metadataValue: Color) :
    MetadataKeyValue<Color>(ItemStackMetadataKey.COLOR, metadataValue)
